packages/pipeline/src/utils/isStageElement.spec.ts

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import isStageElement from './isStageElement';
+import Stage from '../Stage';
+import { Size, Variant } from '../styles';
+
+describe('packages/pipeline/utils/isStageElement', () => {
+  test('returns "true" if the element is a "Stage" element', () => {
+    const props = {
+      size: Size.XSmall,
+      variant: Variant.Default,
+      children: '$legitimate',
+    };
+    const element = React.createElement(Stage, props);
```

Comment:
You should be able to make this file `.tsx` and write this as JSX rather than using `createElement`.

Reply:
Resolved by ba8c9a9